src/providers/bttv/BttvEmotes.cpp

```diff
@@ -86,7 +90,10 @@ namespace {
                         Image::fromUrl(getEmoteLinkV3(id, "2x"), 0.5),
                         Image::fromUrl(getEmoteLinkV3(id, "3x"), 0.25),
                     },
-                    Tooltip{name.string + "<br />Channel BetterTTV Emote"},
+                    Tooltip{name.string + "<br />Channel BetterTTV Emote" +
+                            ((author.string.isEmpty())
+                                 ? "<br />Channel Emote"
+                                 : "<br />By : " + author.string)},
```

Comment:
If no author information is found, we simply shouldn't add any text at all.
Right now, all BTTV Channel Emotes have a tooltip that look like this:
```
FeelsKKonaMan
Channel BetterTTV Emote
Channel Emote
```

Reply:
Should I prevent it from printing anything if 'author' is an empty string? 

Yea now that I look at it it doesn't look good.

That should only be printing for emotes that are uploaded by the user. So emotes that are within the 'channelEmotes'.

For 'sharedEmotes' it looks like this for me. 
![sharedEmotesExample](https://user-images.githubusercontent.com/19235198/83976649-26fc4980-a8b0-11ea-961b-791bed2c3b22.png)

Reply:
Leaving it blank also didn't sit well with me considering that everything else aside from twitch emotes displayed the user that posted the emote (AKA author). 

Since "channelEmotes" contain emotes published by the user of the current channel  (AKA author) I checked to see if the current channels username was stored in the "TwitchChannel" class. This class contains the "loadChannel" function which is used to load the channel emotes among other data. Sure enough, the "userName" is stored within that class I opted to pass it through to load channel and parseChannelEmotes.